Makefile.dep

```diff
@@ -319,38 +313,6 @@ ifneq (,$(filter gnrc_ipv6_nc,$(USEMODULE)))
   USEMODULE += ipv6_addr
 endif
 
-ifneq (,$(filter gnrc_ipv6_nib_6lbr,$(USEMODULE)))
```

Comment:
why do you remove the nib dependency information?

Reply:
Probably rebase error.

Reply:
Fixed